assets/support.rackspace.com/src/css/less/components/search-box.less

```diff
@@ -0,0 +1,58 @@
+[data-carina-search-box] {
```

Comment:
@nimbinatus Rather than doing this manually... https://github.com/ekryski/less2sass